src/components/Inventory/index.tsx

```diff
@@ -209,11 +209,16 @@ export default function Inventory(props: InventoryProps): JSX.Element {
     onApplyFilters();
   }, [filters, onApplyFilters]);
 
+  const onCloseImportInventoryModal = () => {
+    onApplyFilters();
+    setImportInventoryModalOpen(false);
+  };
+
   return (
     <TfMain>
       <ImportInventoryModal
         open={importInventoryModalOpen}
-        onClose={() => setImportInventoryModalOpen(false)}
+        onClose={onCloseImportInventoryModal}
```

Comment:
will be cool to avoid the apply-filter if it is a modal cancel.
for later, we should start introducing the concept of onCancel in our modals perhaps.

Reply:
nice, thanks @constanzauanini :)